eZ/Publish/Core/MVC/Symfony/SignalSlot/AssignSectionSlot.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * File containing the Legacy\AssignSectionSlot class
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
+ */
+
+namespace eZ\Publish\Core\MVC\Symfony\SignalSlot;
+
+use eZ\Publish\Core\SignalSlot\Signal;
+use eZContentCacheManager;
+use eZContentObject;
+use eZSearch;
```

Comment:
Those 3 imports belong to legacy and are not used below (same remark for other slots)